grakn-engine/src/main/java/ai/grakn/engine/controller/response/ConceptBuilder.java

```diff
@@ -80,10 +80,10 @@ public static Things buildThings(ai.grakn.concept.Type type, int offset, int lim
     //TODO: This will scale poorly with super nodes. Need to introduce some sort of paging maybe?
```

Comment:
I think we just did the TODO!

Reply:
Our paging is artificial I think, so i will keep the todo for the moment.

Reply:
oh yeah! I forgot about that.